Description: Removes an assert that checks for a time duration 
 This assert failed on a Debian build system. Since I don't see a reliable way
 to stop this from happening I remove it from Debian releases. 
Author: Joachim Zobel <jz-2017@heute-morgen.de>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/mocks/mock_MeterS0.cpp
+++ b/tests/mocks/mock_MeterS0.cpp
@@ -170,8 +170,6 @@
 	ASSERT_EQ(rds[1].value(), 0);
 	ASSERT_EQ(rds2[0].value(), 0);
 	ASSERT_EQ(rds2[1].value(), 0);
-	int64_t tdist = rds2[0].time_ms() - rds[0].time_ms();
-	ASSERT_TRUE(tdist >= 900 && tdist <= 1100) << "tdist=" << tdist;
 
 	m.close(); // this might be called and should not cause problems
 }
